src/Context/ImageActorContext.js

```diff
@@ -45,10 +45,14 @@ class ImageActorContext {
   // Map of image intensity component to array of [minValue, maxValue] for
   // mapping colors
   colorRanges = new Map()
+  // If not touched, keep growing ranges as new parts of the image are loaded
+  colorRangesTouched = new Map()
```

Comment:
Perhaps name `colorRangesAdjustWithROI` or similar. For very large datasets, the color range of the largest scale is often much smaller than the true range in the data. This can lead to unexpected / undesirable behavior. In these cases, the colorRange of the full-size, full-resolution dataset can be stored as metadata with the image. 

My impression, please clarify in the comments, is that the map is from image intensity component to a boolean.

Reply:
Changed to `colorRangesAutoAdjust` as range can expand automatically on scale changes.